functions/index.js

```diff
@@ -225,3 +225,20 @@ exports.regionSubscribeCreate = functions.region('europe-west1').firestore.docum
       console.log('ID', snap.id);
     }
   });
+
+  exports.reportedPostsCreate = functions.region('europe-west1').firestore.document('/reported-posts/{reportRequestId}')
+  .onCreate(async (snap, context) => {
+    try {
+      const db = admin.firestore();
+      const snapValue = snap.val();
+      const { postId, uid } = snapValue;
```

Comment:
could we rename postId to askForHelpId ?

Reply:
sure! Will adapt in frontend as well